### ISSUE TITLE
Create CI/CD pipeline to upload the package to PyPi

### ISSUE THREAD

**Issue by xinxiao** [zhanyong-wan/dongbei#45]
Create CI/CD pipeline to publish dongbei-lang interpreter binary to PyPi so that users can install the tool with `pip` and directly use dongbei-lang in the following procedure:

```bash
# Install
pip install dongbei-lang

# Run dongbei-lang scripts
dongbei-lang <script>.dongbei
```

**Comment by zhanyong-wan** [comment]
此计甚好。PR来一个？

**Comment by xinxiao** [comment]
最近有点儿忙 预计周三上PR

**Comment by xinxiao** [comment]
@zhanyong-wan 能把这个task assign给我吗

**Comment by zhanyong-wan** [comment]
好啊。这个名字能不能短一点。憋"dongbei-lang"，就"dongbei"行不？

**Comment by xinxiao** [comment]
好的 pip package的名字和cli binary的名字我们都可以code review的时候再具体决定 ;)

**Comment by xinxiao** [comment]
PR整好了 #47

**Comment by xinxiao** [comment]
Issue fixed by #47
